lib/Service/TableTemplateService.php

```diff
@@ -620,33 +710,30 @@ private function createColumn(int $tableId, array $parameters): ?Column {
 
 		return $this->columnService->create(
 
-			// tableId
-			$tableId,
-
-			// title
-			(isset($parameters['title']) && $parameters['title'] != '') ? $parameters['title'] : $this->l->t('No title given'),
-
 			// userId
 			$this->userId,
 
+			// tableId
```

Comment:
Once PHP 8 is minimum required version this could make use of https://stitcher.io/blog/php-8-named-arguments